service-registry/registry-zero-config/src/main/java/org/apache/servicecomb/zeroconfig/client/ClientUtil.java

```diff
@@ -41,86 +39,68 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ClientUtil.class);
 
-  private static ScheduledExecutorService executor = Executors.newSingleThreadScheduledExecutor();
+  public static ClientUtil INSTANCE = new ClientUtil();
 
-  private static Microservice microserviceSelf = new Microservice();
+  private static ServerMicroserviceInstance serviceInstanceForHeartbeat = null;
```

Comment:
This is a sgingletan class, a static member seems a little bit hard to figure out if it is correct.

Reply:
changed to non-static ....